gnd/src/main/java/com/google/android/gnd/ui/common/TwoLineToolbar.java

```diff
@@ -41,12 +41,16 @@ public TwoLineToolbar(Context context, @Nullable AttributeSet attrs) {
     ButterKnife.bind(this);
   }
 
-  public void setTitle(String title) {
+  public void setTitle(@Nullable String title) {
     titleText.setText(title);
   }
 
-  public void setSubtitle(String subtitle) {
+  public void setSubtitle(@Nullable String subtitle) {
     subtitleText.setText(subtitle);
-    subtitleText.setVisibility(subtitle.isEmpty() ? View.GONE : View.VISIBLE);
+    if (subtitle == null) {
```

Comment:
To avoid the ternary operator below you could change this to `if(subtitle == null || subtitle.isEmpty()){ // GONE } else { // VISIBLE }`.